doc/sphinx/language/core/assumptions.rst

```diff
@@ -155,6 +155,12 @@ has type :n:`@type`.
    is closed, the :n:`@ident`\(s) become undefined and every object depending on them will be explicitly
    parameterized (i.e., the variables are *discharged*).  See Section :ref:`section-mechanism`.
 
+   :n:`:>`
+     If specified, the name of this field is automatically
+     declared as a coercion from the record name to the class of this
+     field type. Note that fields always verify the uniform
+     inheritance condition.
```

Comment:
This description is too specific. `:>` may be used in other contexts than records. Example:

```
Parameter foo :> nat -> Prop.
Print Coercions. (* foo *)
```

Reply:
I removed "from the record name"--does that cover it?  I can't really say I understand `:>`.

Reply:
Yes, it looks good now.